.gitignore

```diff
@@ -1 +1,3 @@
 node_modules
+.DS_Store
+.gitignore
```

Comment:
are you gitignoring your gitignore?

Reply:
Yup, but I realize that doesn't do anything.. lol